Modix/IEmoteTypeReader.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Discord;
+using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Modix
+{
+    public class IEmoteTypeReader : TypeReader
```

Comment:
I know the intention is that "this is a `TypeReader` that reads `IEmote` values", but I think it's still better to not put `I` at the beginning of a `class` name.